### ISSUE TITLE
Using templated META file, the generated `dune-package` is wrong

### ISSUE THREAD

**Issue by Kakadu** [ocaml/dune#3746]
## Expected Behavior
dune-package is OK

## Actual Behavior
It is in wrong format

````
dune build src
File "/home/kakadu/.opam/4.10.0+fp+flambda/lib/GT/dune-package", line 3, characters 1-8:
3 | (version 8f75697-dirty)
     ^^^^^^^
Error: Unknown field version
````

## Reproduction

1. Install a package with `META.PACKAGENAME.tempalte` present
2. Compile something that depends on your package

## Workaround:

```Makefile

install:
	dune build @install
	dune install
	echo "(lang dune 2.7)" > `ocamlfind query PACKAGENAME`/dune-package
	echo "(use_meta)" >> `ocamlfind query PACKAGENAME`/dune-package
	dune describe --only-packages=PACKAGENAME >> `ocamlfind query PACKAGENAME`/dune-package

``` 

I discussed that yesterday with @rgrinberg 

P.S. OK, maybe workaround is not actually a valid workaround.

**Comment by rgrinberg** [comment]
Fixed by #3743

**Comment by Kakadu** [comment]
It seems that my issue is really fixed now. Thanks!
